Configurations.md

```diff
@@ -0,0 +1,1760 @@
+- **Possible values**: `"Default"`, `"Rfc"`
+
+⚠️ `generics_style` currently has no effect.
+
+## `hard_tabs`
+
+Use tab characters for indentation, spaces for alignment
+
+- **Default value**: `false`
+- **Possible values**: `true`, `false`
+
+#### `false`:
+
+```rust
+fn lorem() -> usize {
+    42 // spaces before 42
+}
+```
+
+#### `true`:
+
+```rust
+fn lorem() -> usize {
+	42 // tabs before 42
+}
+```
+
+See also: [`tab_spaces`](#tab_spaces).
+
+## `impl_empty_single_line`
+
+Put empty-body implementations on a single line
+
+- **Default value**: `true`
+- **Possible values**: `true`, `false`
+
+#### `false`:
+
+```rust
+impl Lorem {
+}
+```
+
+#### `true`:
+
+```rust
+impl Lorem {}
+```
+
+See also [`item_brace_style`](#item_brace_style).
+
+## `indent_match_arms`
+
+Indent match arms instead of keeping them at the same indentation level as the match keyword
+
+- **Default value**: `true`
+- **Possible values**: `true`, `false`
+
+#### `false`:
+
+```rust
+match lorem {
+Lorem::Ipsum => (),
+Lorem::Dolor => (),
+Lorem::Sit => (),
+Lorem::Amet => (),
+}
+```
+
+#### `true`:
+
+```rust
+match lorem {
+    Lorem::Ipsum => (),
+    Lorem::Dolor => (),
+    Lorem::Sit => (),
+    Lorem::Amet => (),
+}
+```
+
+See also: [`match_block_trailing_comma`](#match_block_trailing_comma), [`wrap_match_arms`](#wrap_match_arms).
+
+## `item_brace_style`
+
+Brace style for structs and enums
+
+- **Default value**: `"SameLineWhere"`
+- **Possible values**: `"AlwaysNextLine"`, `"PreferSameLine"`, `"SameLineWhere"`
+
+#### `"AlwaysNextLine"`:
+
+```rust
+struct Lorem
+{
+    ipsum: bool,
+}
+
+struct Dolor<T>
+    where T: Eq
+{
+    sit: T,
+}
+```
+
+#### `"PreferSameLine"`:
+
+```rust
+struct Lorem {
+    ipsum: bool,
+}
+
+struct Dolor<T>
+    where T: Eq {
+    sit: T,
+}
+```
+
+#### `"SameLineWhere"`:
+
+```rust
+struct Lorem {
+    ipsum: bool,
+}
+
+struct Dolor<T>
+    where T: Eq
+{
+    sit: T,
+}
+```
+
+## `match_block_trailing_comma`
+
+Put a trailing comma after a block based match arm (non-block arms are not affected)
+
+- **Default value**: `false`
+- **Possible values**: `true`, `false`
+
+#### `false`:
+
+```rust
+match lorem {
+    Lorem::Ipsum => {
+        println!("ipsum");
+    }
+    Lorem::Dolor => println!("dolor"),
+}
+```
+
+#### `true`:
+
+```rust
+match lorem {
+    Lorem::Ipsum => {
+        println!("ipsum");
+    },
+    Lorem::Dolor => println!("dolor"),
+}
+```
+
+See also: [`indent_match_arms`](#indent_match_arms), [`trailing_comma`](#trailing_comma), [`wrap_match_arms`](#wrap_match_arms).
+
+## `max_width`
+
+Maximum width of each line
+
+- **Default value**: `100`
+- **Possible values**: any positive integer
+
+See also [`error_on_line_overflow`](#error_on_line_overflow).
+
+## `newline_style`
+
+Unix or Windows line endings
+
+- **Default value**: `"Unix"`
+- **Possible values**: `"Native"`, `"Unix"`, `"Windows"`
+
+## `normalize_comments`
+
+Convert /* */ comments to // comments where possible
```

Comment:
Looking at the code it should, but I just checked with

```
// Lorem ipsum:
fn dolor() {}

/* sit amet: */
fn consectetur() {}

/** adipiscing: */
fn elit() {}
```

and it doesn't seem to touch the latter at all. Heck, `fn rewrite_comment` doesn't even seem to get called for it. 🤔